### DISCUSSION TITLE
Infinite loading fix 3.5

### DISCUSSION THREAD

**Discussion by mRama3415** [Raytwo/ARCropolis#365] [General]
It says that 3.5 has fixed the infinite loading issue that was being experienced. If this was due to a large mods folder in the past (17GB+), does this mean that it wont occur anymore on large mods folders? Or should we still keep our mod folders under that range of size? Thanks!

**Comment by jozz024** [comment]
you are correct, it was tested with a 17+ gb modpack and no infinite loads occoured

**Reply by mRama3415** [reply]
awesome! Thanks a lot
